modules/database/src/std/link/links.dbd.pod

```diff
@@ -81,7 +81,7 @@ link(calc, lnkCalcIf)
 =head3 Calculation Link C<"calc">
 
 A calculation link is an input link that can evaluate mathematical expressions
-on scalar (double-precision floating-point) values obtained from up to 12 child
+on scalar (double-precision floating-point) values obtained from up to 21 child
```

Comment:
I think there should be a note about history each time the specific number of links/inputs/... is mentioned.  eg. "21 child input links (12 before UNRELEASED)".  or in other doxygen contexts:

```
@since UNRELEASED Increase number of arguments from 12 to 21.
```

Alternately, when practical instead of mentioning the specific number reference `CALCPERFORM_NARGS`.  Practicality probably means for C APIs, where pre-processor macros are directly usable.

Reply:
In documentation, `CALCPERFORM_NARGS` is a bit impractical because it would force the user to read the source code in order to understand the documentation. Maybe in documentation for developers using the calc functions, but not for the system integrator who wants to know which links they can use.
Or can we expand macros like `CALCPERFORM_NARGS` when building the documentation? Maybe we could implement that for numbers like 21, but can we do that for field ranges like `INPA-INP(A+CALCPERFORM_NARGS-1)`? Probably quite difficult.
I had planned to mention this in the RELEASE notes, waiting for #628.